dev/com.ibm.ws.microprofile.config_fat/test-applications/dynamicSources.war/src/com/ibm/ws/microprofile/appConfig/dynamicSources/test/DynamicSourcesTestServlet.java

```diff
@@ -170,4 +175,48 @@ public void testDynamicUserAddedSources() throws Exception {
         }
     }
 
+    @Test
+    public void testClosedConfigStopsRefreshing() throws Exception {
+        // Remember that the mimimum refresh is currently 500
+        System.setProperty(DYNAMIC_REFRESH_INTERVAL_PROP_NAME, "" + 1);
+        ConfigBuilder b = ConfigProviderResolver.instance().getBuilder();
+
+        MySource s1 = new MySource("MySource S1");
+        s1.setOrdinal(600);
+        s1.put("1", "1");
+
+        b.withSources(s1);
+        b.addDefaultSources();
+        Config config = b.build();
+
+        try {
+            TestUtils.assertContains(config, "1", "1");
+            TestUtils.assertContains(config, "server_env_property", "server.env.value");
+
+            long lastRefresh = s1.lastRefresh;
+
+            // Remember that the mimimum refresh is currently 500 ... so wait longer than that
+            Thread.sleep(1500);
```

Comment:
maybe make this ConfigConstants.MINIMUM_DYNAMIC_REFRESH_INTERVAL + 1000 ?

Reply:
Done